apps/frontend/src/app/components/header/header.component.html

```diff
@@ -24,9 +24,17 @@ <h1 class="title" routerLink="/">Verseghy</h1>
         <a (click)="drawer = !drawer" href="" routerLink="/misc/authors">Fejlesztők</a>
       </div>
     </div>
-    <a class="ekreta" href="https://klik035992001.e-kreta.hu/Adminisztracio/Login" target="_blank">
-      <img alt="" src="assets/icons/kreta2.png" />
-    </a>
+    <div (click)="kretaSubmenu = !kretaSubmenu" [ngClass]="{ open: kretaSubmenu }" class="menu ekreta">
+      <span><img alt="" src="assets/icons/kreta2.png"/></span> <i class="material-icons arrow">expand_more</i>
```

Comment:
I feel like this span tag is unnecessary, we use span tags to group inline elements or for displaying text, not for holding images.

Reply:
Fixed in 2239f82